easybuild/easyconfigs/l/libunwind/libunwind-1.4.0-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,44 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 	https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+# License::	MIT
+#
+# Notes::
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'libunwind'
+version = '1.4.0'
+
+homepage = 'https://www.nongnu.org/libunwind/'
+description = """The primary goal of libunwind is to define a portable and efficient C programming interface
+ (API) to determine the call-chain of a program. The API additionally provides the means to manipulate the
+ preserved (callee-saved) state of each call-frame and to resume execution at any point in the call-chain
+ (non-local goto). The API supports both local (same-process) and remote (across-process) operation.
+ As such, the API is useful in a number of applications"""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = [GNU_SAVANNAH_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['df59c931bd4d7ebfd83ee481c943edf015138089b8e50abed8d9c57ba9338435']
+
+builddependencies = [('binutils', '2.34', '', True)]
+
+parallel = 1
+
+dependencies = [
+    ('XZ', '5.2.5'),
+]
+
+preconfigopts = 'export LIBS="$LIBS -llzma" && export CFLAGS="$CFLAGS -fuse-ld=bfd" && '
```

Comment:
The LIBS= part should no longer be required (actually for a couple versions already) To be sure adding `configopts = '--enable-minidebuginfo'` would be reasonable